src/SFA.DAS.EAS.Portal/DependencyResolution/ProviderServices.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using SFA.DAS.EAS.Portal.Configuration;
+using SFA.DAS.Providers.Api.Client;
+
+namespace SFA.DAS.EAS.Portal.DependencyResolution
+{
+    public static class ProviderServices
+    {
+        public static IServiceCollection AddProviderServices(this IServiceCollection services, HostBuilderContext hostBuilderContext)
+        {
+            var apiClientConfig = hostBuilderContext.Configuration.GetSection<ApprenticeshipInfoServiceApiConfiguration>(ConfigurationKeys.ApprenticeshipInfoServiceApi);
+
+            services.AddTransient<IProviderApiClient>(sp => new ProviderApiClient(apiClientConfig.BaseUrl));
+            // if we want to support config changes on the fly, we could create wrapper class that accepts IOptionsMonitor and picks out url
```

Comment:
Do we need these comments?  Create a tech debt item instead rather than litter the code with your thoughts.

Reply:
Litter removed! :boom: